<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-S3E6FN1T" ignored="0" name="Login_data-driven" tests="5" failures="3" timestamp="2019-12-03T20:15:18 EST" time="9.543" errors="0">
  <testcase name="asign" time="5.326" classname="datadriventestcases.LoginDataDriven"/>
  <testcase name="login" time="0.435" classname="datadriventestcases.LoginDataDriven">
    <failure type="java.lang.AssertionError" message="expected [Zero - Log in] but found [Zero - Account Summary]">
      <![CDATA[java.lang.AssertionError: expected [Zero - Log in] but found [Zero - Account Summary]
at datadriventestcases.LoginDataDriven.login(LoginDataDriven.java:29)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="login" time="0.383" classname="datadriventestcases.LoginDataDriven">
    <failure type="java.lang.AssertionError" message="expected [Zero - Log in] but found [Zero - Account Summary]">
      <![CDATA[java.lang.AssertionError: expected [Zero - Log in] but found [Zero - Account Summary]
at datadriventestcases.LoginDataDriven.login(LoginDataDriven.java:29)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="login" time="0.323" classname="datadriventestcases.LoginDataDriven">
    <failure type="java.lang.AssertionError" message="expected [Zero - Log in] but found [Zero - Account Summary]">
      <![CDATA[java.lang.AssertionError: expected [Zero - Log in] but found [Zero - Account Summary]
at datadriventestcases.LoginDataDriven.login(LoginDataDriven.java:29)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="login" time="1.411" classname="datadriventestcases.LoginDataDriven"/>
</testsuite> <!-- Login_data-driven -->
